File: src/SM.Home.UI/src/services/account-service/models/account-create-request.ts
export class AccountCreateRequest {
    
    username: string | null | undefined;
    email: string | null | undefined;
    encryptedPassword: string | null | undefined;
    ivHex: string | null | undefined;

    constructor(
        username: string | null | undefined,
        email: string | null | undefined,
        encryptedPassword: string | null | undefined,
        ivHex: string | null | undefined) {
        this.username = username;
        this.encryptedPassword = encryptedPassword;
        this.email = email;
        this.ivHex = ivHex;
    }

}

File: src/IdentityApi/SM.Identity.API.Client/Models/Account/AccountCreateRequest.cs
using System.Text.Json.Serialization;

namespace SM.Identity.API.Client.Models.Account
{
    public class AccountCreateRequest
    {
        [JsonPropertyName("username")]
        public string UserName { get; set; }

        [JsonPropertyName("email")]
        public string Email { get; set; }

        [JsonPropertyName("encrryptPassword")]
        public string EncrryptPassword { get; set; }

        [JsonPropertyName("ivHex")]
        public string IvHex { get; set; }
    }
}
